libs/libcommon/src/libcommon/queue.py

```diff
@@ -116,6 +119,21 @@ class JobQueryFilters(TypedDict, total=False):
     difficulty__lte: int
 
 
+PA_SCHEMA = Schema(
+    {
+        "_id": bson.ObjectId,
```

Comment:
https://www.mongodb.com/docs/languages/python/pymongo-arrow-driver/current/data-types/ for more reference